api/rest/execInit.go

```diff
@@ -52,7 +52,7 @@ func HandleInit(c *gin.Context) {
 		return
 	}
 
-	err := HandleKubeConfigCreation(c, &initConfigParams, token)
+	err := HandleKubeConfigCreation(&initConfigParams.KubeConfigParams, token, execRequest.ContainerName)
```

Comment:
I believe it could work in more optimal way.
execRequest already has pod name, while `HandleKubeConfigCreation` resolves it again in https://github.com/eclipse/che-machine-exec/blob/35dff49aa0ac5db3d847569a24e9f97ff7b7bf66/filter/kubernetes_filter.go#L104

So, the whole ContainerInfo could be propagated instead of just container name.

Reply:
The issue is that the other flow (`execKubeConfig`) currently doesn't do the resolve step and uses containerName from `initConfigParams`, so we would have to change that process as well.